### ISSUE TITLE
构建 graph 问题探讨

### ISSUE THREAD

**Issue by chenluda** [SsGood/MMGL#3]
恭喜！非常精彩的工作！

这里探讨个问题：

代码里似乎是将 train set 和 test set 统一输入构建 graph，train phase 中 test set 不参与 bp。

如果是这样，那么论文中的 

_“ the unseen patient u is first added to the existing population graph by MARL with AGL. ... It should be noting that the test patients are not kept in the graph in the testing phase. ”_ 

该怎么理解呢？

期待回复，谢谢！

**Comment by SsGood** [comment]
首先感谢您对我们工作的关注！我们当前开源的版本是transductive learning的版本，即您提到的

> 代码里似乎是将 train set 和 test set 统一输入构建 graph，train phase 中 test set 不参与 bp。

论文中的描述是inductive learning的版本，该版本的代码我们将在10月份开源，请您持续关注，谢谢！

Thanks for your attention to our work!  The current repository is a PyTorch transductive version of the MMGL model as proposed in our paper, the inductive version will be coming soon in October, please stay tuned, thank you!

**Comment by chenluda** [comment]
收到，感谢您的回复！
